docs/09_tutorials/10_nested_containers.md

```diff
@@ -0,0 +1,199 @@
+---
+content_title: Nested Containers
+---
+
+## Overview
+
+This tutorial demonstrates the usage of various C++ nested-containers in smart contracts for the following EOSIO types:
+
+* [Multi-Index Table](https://developers.eos.io/manuals/eosio.cdt/latest/how-to-guides/multi-index/index)
+* [Key-Value Map](https://developers.eos.io/manuals/eosio.cdt/latest/how-to-guides/key-value-api/kv_map/index)
+* [Key-Value Table](https://developers.eos.io/manuals/eosio.cdt/latest/how-to-guides/key-value-api/kv_table/index) 
+
```

Comment:
kv_table is NOT yet involved, only kv_map is under investigation  in the nested containers project